django/db/migrations/state.py

```diff
@@ -29,6 +29,29 @@ def _get_app_label_and_model_name(model, app_label=''):
         return model._meta.app_label, model._meta.model_name
 
 
+def _get_related_models(m):
+    """
+    Returns all model that have a direct relationship to the given model.
+
+    This is a private API used by `get_related_models_recursive` below.
+    """
```

Comment:
Added a small docstring since this is really just a private API used by `get_related_models_recursive`.